examples/solvers/admm_spectral_tomography.py

```diff
@@ -0,0 +1,169 @@
+"""Total variation spectral tomography using preconditioned nonlinear ADMM.
```

Comment:
Examples like this truly need to go into a "complicated" folder so to say, this is getting out of hand.

Nothing against the example per se, just that we want beginners to find them "in the right order".